README.md

```diff
@@ -48,13 +48,13 @@
 
 [JSON:API](http://jsonapi.org) specification
 [reached 1.0 on 29th May 2015](http://www.programmableweb.com/news/new-json-api-specification-aims-to-speed-api-development/2015/06/10)
-and we also believe it is a big day for RESTful API-s as this specification can help you to make APIs more robust and future-proof. Woohoo Labs. Yin (named after Yin-Yang) was born to bring efficiency and elegance for your JSON:API servers.
+and we also believe it is a big day for RESTful APIs as this specification can help you to make APIs more robust and future-proof. Woohoo Labs' Yin (named after Yin-Yang) was born to bring efficiency and elegance to your JSON:API servers.
 
 #### Features
 
 - 100% [PSR-7](http://www.php-fig.org/psr/psr-7/) compatibility
 - 99% [JSON:API 1.0](http://jsonapi.org/) compatibility (approximately)
-- Developed for efficiency and ease of use
+- Developed for efficiency and ease-of-use
```

Comment:
When I wrote this, I check it on Google and "ease of use" was way more common. Do you always use this expression with hyphen?